utl/vw-important-features

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python3
+# ==========================================================================================
+# Find important features
+# ==========================================================================================
+
+'''
+Usage example:
+>python vw-important-features -d data.json --ml_args "--cb_adf -l 0.01 --cb_type mtr"
```

Comment:
If the file is made executable (`chmod +x vw-important-features`) before check-in, there's no need to prefix the call with `python`